sos/plugins/convert2rhel.py

```diff
@@ -0,0 +1,27 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+
+
+class convert2rhel(Plugin, RedHatPlugin):
+
+    plugin_name = 'convert2rhel'
+    profiles = ('system')
+    verify_packages = ('convert2rhel$',)
+
```

Comment:
Worth to be called only when `convert2rhel` package is present? I.e. `packages = ('convert2rhel')` ?

Reply:
True, let me add that, and remove the unused import, and resend.